requirements-dev.txt

```diff
@@ -10,13 +10,14 @@ pylint==2.16.2
 yapf==0.32.0
 
 # Doc Build
-sphinx==6.1.3
+sphinx~=6.2.1
```

Comment:
FYI this means this is equivalent to `>=6.2.1,<6.3`. 

It's great that you're pinning every dep! I wish more projects did that. But I generally prefer `~=` so that you can pull in bug fixes.